### ISSUE TITLE
vscode does not provide intellisense

### ISSUE THREAD

**Issue by xxc-zsz** [microsoft/pylance-release#3249]
The VSCode autocomplete option doesn't work for **tensorflow** and **keras** libraries; However i've installed python and pylance extension on it.

Here i'm trying to use `layers` or `preprocessing` from `keras` API but it doesn't show anything at all.

![image](https://user-images.githubusercontent.com/108051416/186876556-c9b1681f-f7a2-4990-a7ed-85953e01ab78.png)

![image](https://user-images.githubusercontent.com/108051416/186876703-e50110d5-15e3-4be1-8a21-1378a1e2bb84.png)

I used a virtual environment and made sure to have the **tensorflow** package installed in the environment.

I also set these for vscode, but it doesn't seem to work.

```
    "python.analysis.extraPaths": [
        "C:\\WorkSpace\\PyTest0802\\.venv\\Lib\\site-packages"
    ],
    "python.autoComplete.extraPaths": [
        "C:\\WorkSpace\\PyTest0802\\.venv\\Lib\\site-packages"
    ],
```

**Comment by bschnurr** [comment]
shouldn't have to set extra paths.

are you setting your environment? see https://code.visualstudio.com/docs/python/environments#_select-and-activate-an-environment

**Comment by xxc-zsz** [comment]
Sorry that my statement might not be clear enough.
Adding an extra path setting is just a workaround I've found, it just still doesn't solve the problem.
Without setting the extra path, intellisense just doesn't work like the picture I showed.
I'm not setting up the environment. I'm asking how to make intellisense work correctly. I have my environment set up. Create and activate a virtual environment, choose an interpreter for the virtual environment, and install these packages for the virtual environment. I have done all of these.
